test/e2e/framework/kcp.go

```diff
@@ -136,6 +137,11 @@ func WithLogStreaming(o *runOptions) {
 // Run runs the kcp server while the parent context is active. This call is not blocking,
 // callers should ensure that the server is Ready() before using it.
 func (c *kcpServer) Run(parentCtx context.Context, opts ...RunOption) error {
+	path, err := exec.LookPath("kcp")
+	if err != nil {
+		return fmt.Errorf("kcp binary not found")
```

Comment:
Maybe embed the error in this message?

Reply:
yeah

Reply:
```
kcp binary not found: exec: "kcp": executable file not found in $PATH
```

Reply:
I think I can return the error directly, the message is redundant